internal/handlers/handlers.go

```diff
@@ -1 +1,76 @@
 package handlers
+
+import (
+	"io"
+	"log"
+	"net/http"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/Yandex-Practicum/go1fl-sprint6-final/internal/service"
+)
+
+
+func MainHandler(w http.ResponseWriter, req *http.Request) {
+    if req.Method != http.MethodGet {
+        http.Error(w, "Сервер не поддерживает "+req.Method, http.StatusMethodNotAllowed)
+        return
+    }
+	path := ""
+	for _, path = range []string{"index.html","../index.html"} {
+		if _, err := os.Stat("index.html"); err == nil {
+			break
+		}
+	}
+    http.ServeFile(w, req, path)
+}
+
+
+func UploadHandler(w http.ResponseWriter, req *http.Request) {
+    if req.Method != http.MethodPost {
+        http.Error(w, "Сервер не поддерживает "+req.Method, http.StatusMethodNotAllowed)
+        return
+    }
+
+	err := req.ParseMultipartForm(0)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
```

Comment:
это проблема клиента
```suggestion
		http.Error(w, err.Error(), http.StatusBadRequest)
```

Reply:
Исправлено.